packages/synapse-react-client/test/utils/hooks/SynapseAPI/useEntityBundle.test.tsx

```diff
@@ -1,4 +1,4 @@
-import { renderHook } from '@testing-library/react-hooks'
+import { renderHook } from '@testing-library/react'
```

Comment:
Functionality in @testing-library/react-hooks has been ported to @testing-library/react, so we dropped @testing-library/react-hooks